src/assets/projectJSS/Colors.ts

```diff
@@ -0,0 +1,16 @@
+// File contains all theme colors through the project, documented in 1st page of PowerPoint/Pdf
+
+// Usage: color: `${NAV_BAR_COLOR}`,
+
+export const NAV_BAR_COLOR = 'rgb(40,54,67)';
+export const LIGHT_COLOR_TEXT = '#FFFFFF';
+export const DARK_TEST_COLOR = '#595959'; // For advertisments mostly
+export const AVATAR_COLOR = 'rgb(73,90,114)';
+export const TOOLTIP_OR_SCROLLER_COLOR = 'rgb(127, 140, 156)';
+export const TEXT_AND_ICONS_ADS_COLOR = 'rgb(46, 188, 145)'; // Advertisements
+export const TEXT_AND_ICONS_ADS_BACKGROUND_COLOR = '#F8F9FC'; // Advertisements
+export const NEW_JOB_COLOR = '#21BD90';
+export const YELLOW_STAR_COLOR = 'rgb(254,201,62)'; // Same as in PowerPoint
+
+// Shadow similar to PowerPoint
+export const BOX_SHADOW = "0px 1px 7px 1px rgba(0,0,0,0.75)";

```

Comment:
IMO it's better to have a jss like object that you can spread.  
For example:
```jsx
const lightShadow = {
    boxShadow: '0px 1px 7px 1px rgba(0,0,0,0.75)'
};

// ... the component's jss class
{
adContainer: {
   ...lightBoxShadow
},
```

Reply:
Done 😄